src/Features/CSharp/Portable/SignatureHelp/AbstractOrdinaryMethodSignatureHelpProvider.cs

```diff
@@ -41,7 +41,8 @@ internal static SignatureHelpItem ConvertMethodGroupMethod(
             GetSeparatorParts(),
             GetMethodGroupPostambleParts(),
             [.. method.Parameters.Select(p => Convert(p, semanticModel, position, documentationCommentFormattingService))],
-            descriptionParts: descriptionParts);
+            descriptionParts: descriptionParts,
+            static symbol => symbol is null ? null : SymbolDisplay.ToDisplayString(symbol, SymbolDisplayFormat.MinimallyQualifiedFormat));
```

Comment:
need to pass this along as we are now synthesizing symbols, and synthesized symbols don't support .ToDisplayString being called directly on them (as they don't belong to any language).